templates/contactList.html

```diff
@@ -13,4 +13,7 @@
 			<h2>{{ctrl.t.emptySearch}}</h2>
 		</div>
 	</div>
+	<div ng-if="filtered.length>0" class="contacts-list-count">
+		{{ filtered.length }} {{ filtered.length==1 ? ctrl.t.oneContact : ctrl.t.multipleContacts }}
```

Comment:
in pure js we would call n() instead of t() - but with angular this is a bit different.

@raimund-schluessler @georgehrke what is your approach with respect to plural strings in angular and ownCloud? THX